### ISSUE TITLE
Is this still working?

### ISSUE THREAD

**Issue by cetso** [binary1230/wellsfargo-bulk-PDF-statement-downloader#13]
Hi,

I need to see every transaction from my Wells Fargo account. I followed the steps, but I didn't see the download button.

Thank you

**Comment by ajquick** [comment]
Still works! I just had to make sure my ad blocker was turned off.

**Comment by EthanSchweizerTR** [comment]
Yep it works! I did notice you have to select the account and time period for the big ass buttons to show up at the bottom. It doesnt download "all" statements in one click. You can only download statements for specific periods but its SOOO much better than the default single download

**Comment by razorfrog** [comment]
Just worked for me!

**Comment by ajquick** [comment]
Not working any longer for me. :(

Any luck for anyone else?

**Comment by John-AZ** [comment]
Still working for me. Great tool.

**Comment by bcodemz** [comment]
I absolutely struggled to get this to work. I disabled all extensions and it still doesn't work. I finally got it to work by downloading Chromium so I have a completely fresh version of Chrome, and even then it didn't work right away. For some reason the first time I clicked on a year it didn't work, I had to click on a few different years on the drop down menu to get it working.

Nevertheless, thank you for this amazing tool!

**Comment by ishway** [comment]
Like others, I had a few challenges to get it to work, but I eventually succeeded.  Here is what I did:

1. Disabled my add blocker for the site and page (I use AdblockPlus).
2. Ran the JS code from my bookmark (I had to run this AGAIN after disabling the add blocker).
3. Selected a specific year in the "For time period" pulldown (I had already selected the account in the other one)
4. Then voila!  Two large buttons appeared on the page, a red one on the left that says "Cancel" and a green one on the right that says "Download Statements".
5. Pressing "Download Statements" downloaded every statement shown on the page for the selected time period.
6. Once it completed, the buttons disappeared, but upon selecting a new time period, they appeared again, and I was able to continue.

**Comment by gitoff** [comment]
Still working. Used Brave. Disabled Brave Shields.
